PlaceRouteHierFlow/placer/ILP_solver.cpp

```diff
@@ -238,16 +248,17 @@ double ILP_solver::GenerateValidSolution(design& mydesign, SeqPair& curr_sp) {
 
   // set_add_rowmode(lp, FALSE);
   {
+    double row[N_var] = {0};
```

Comment:
The objective should be area: max(xi)*max(yi) and HPWL: \sum_{i\in net}(max(x_i)+max(y_i))

Reply:
HPWL is added in the latest commit.

Reply:
area is also added.